.github/workflows/pythonapp.yml

```diff
@@ -43,6 +44,48 @@ jobs:
         poetry install
         poetry run pytest tests
 
+  gsutil-upload-staging:
+    if: github.ref == 'refs/heads/staging'
+    name: Upload static Staging
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@master
+
+      - uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+        with:
+          version: '270.0.0'
+          service_account_email: ${{ secrets.SA_EMAIL }}
+          service_account_key: ${{ secrets.GOOGLE_APPLICATION_CREDENTIALS }}
+
+      - run: |
+             gcloud auth configure-docker
```

Comment:
Do you think this is needed? It's for the docker auth, that's useful for building/pushing to docker registry.

Reply:
Frankly, do not know. Seemed important :)

Reply:
I would guess that the service account key specification in above could be enough. Would take this out and see